tests/testthat/test-autoplot.R

```diff
@@ -329,12 +329,21 @@ test_that("plot_perf_vs_iter with fairness metrics (#773)", {
 
 test_that("regular grid plot", {
   skip_if_not_installed("ggplot2", minimum_version = "3.5.0")
```

Comment:
Was this test previously wrongly described? i.e. we weren't actually testing against `regular_grid()` before these changes, were we?

Reply:
Kind of. A one-point design is the same for any one-parameter design with one design point. I changed since the pre-optimized designs don't work in this situation. I'll put an issue in for that. @hfrick